weldx/transformations/local_cs.py

```diff
@@ -889,14 +895,15 @@ def interp_time(
         if (not self.is_time_dependent) or (time is None):
             return self
 
-        # use LCS reference time if none provided
-        if isinstance(time, LocalCoordinateSystem) and time_ref is None:
-            time_ref = time.reference_time
-        time = ut.to_pandas_time_index(time)
+        # handle LCS as time
+        if isinstance(time, LocalCoordinateSystem):
+            if time_ref is None:
+                time_ref = time.reference_time
+            time = time.time
```

Comment:
This should be revisited if we add a `TimeDependent` metaclass. `Time` should be capable to handle it on its own

Reply:
Done in #464